lib/src/tar/tar_file.dart

```diff
@@ -118,7 +116,7 @@ class TarFile {
     return _content;
   }
 
-  List<int> get contentBytes => content as List<int>;
+  Uint8List get contentBytes => content as Uint8List;
```

Comment:
I've been meaning to switch back to explicit Uint8List instead of the List<int> interface class, which seemed to have caused some confusion.